pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/PackageSettings.java

```diff
@@ -91,14 +91,22 @@
     List<String> getSupportedLanguageVersions(T t);
 
     /**
-     * Determines if the package requires a build from source.
```

Comment:
Let's keep the descriptive word here "Determines" because that's Javadoc style -- this function "determines ...".
In some other languages the imperative style is preferred as if giving a command to the function: "Determine ..."
Similarly below, using "Checks ... " is probably more fitting to this style.